<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>oggConfigPage</class>
 <widget class="QWidget" name="oggConfigPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>519</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ogg encoding configuration</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="oggUseQualityCheck">
     <property name="text">
      <string>Use quality setting (recommended)</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QSlider" name="oggQualitySlider">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimum">
        <number>-10</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="singleStep">
        <number>5</number>
       </property>
       <property name="pageStep">
        <number>10</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksAbove</enum>
       </property>
       <property name="tickInterval">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="oggQualitySpin">
       <property name="decimals">
        <number>1</number>
       </property>
       <property name="minimum">
        <double>-1.000000000000000</double>
       </property>
       <property name="maximum">
        <double>10.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="oggQualityLabel">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string notr="true">(~160kbps)</string>
       </property>
       <property name="buddy">
        <cstring>oggQualitySlider</cstring>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="oggBitrateGroup">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Use bitrate</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="formLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Minimal bitrate:</string>
        </property>
        <property name="buddy">
         <cstring>oggMinBitrateCbx</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="oggMinBitrateCbx">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Sets minimum bitrate (in kb/s).</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Nominal bitrate:</string>
        </property>
        <property name="buddy">
         <cstring>oggNormBitrateCbx</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="oggNormBitrateCbx">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Sets target bitrate (in kb/s).</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Maximum bitrate:</string>
        </property>
        <property name="buddy">
         <cstring>oggMaxBitrateCbx</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="oggMaxBitrateCbx">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Sets maximum bitrate (in kb/s).</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="oggGainGroup">
     <property name="title">
      <string>ReplayGain</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QGridLayout" name="formLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="oggGainLabel">
          <property name="text">
           <string>Calculate gain:</string>
          </property>
          <property name="buddy">
           <cstring>oggGainCbx</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="oggGainCbx"/>
        </item>
        <item row="0" column="2">
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>122</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>oggUseQualityCheck</tabstop>
  <tabstop>oggQualitySlider</tabstop>
  <tabstop>oggQualitySpin</tabstop>
  <tabstop>oggMinBitrateCbx</tabstop>
  <tabstop>oggNormBitrateCbx</tabstop>
  <tabstop>oggMaxBitrateCbx</tabstop>
  <tabstop>oggGainCbx</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>oggUseQualityCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>oggBitrateGroup</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>369</x>
     <y>58</y>
    </hint>
    <hint type="destinationlabel">
     <x>232</x>
     <y>137</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>oggUseQualityCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>oggQualitySlider</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>369</x>
     <y>58</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>32</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>oggUseQualityCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>oggQualitySpin</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>369</x>
     <y>58</y>
    </hint>
    <hint type="destinationlabel">
     <x>439</x>
     <y>29</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>oggUseQualityCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>oggQualityLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>369</x>
     <y>58</y>
    </hint>
    <hint type="destinationlabel">
     <x>513</x>
     <y>32</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>oggUseQualityCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_2</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>306</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>111</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>oggUseQualityCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_3</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>256</x>
     <y>58</y>
    </hint>
    <hint type="destinationlabel">
     <x>111</x>
     <y>144</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>oggUseQualityCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_4</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>214</x>
     <y>58</y>
    </hint>
    <hint type="destinationlabel">
     <x>111</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>oggUseQualityCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>oggMinBitrateCbx</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>349</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>193</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>oggUseQualityCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>oggNormBitrateCbx</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>397</x>
     <y>58</y>
    </hint>
    <hint type="destinationlabel">
     <x>193</x>
     <y>144</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>oggUseQualityCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>oggMaxBitrateCbx</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>459</x>
     <y>58</y>
    </hint>
    <hint type="destinationlabel">
     <x>193</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
